utils/encrypt.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import (
+	"crypto/sha1"
+	"encoding/base64"
+)
+
+func HashSHA256(stringToHash string) string {
+	hasher := sha1.New()
+	hasher.Write([]byte(stringToHash))
+	return base64.URLEncoding.EncodeToString(hasher.Sum(nil))
+}
```

Comment:
Consider using bcrypt for encryption.

Reply:
Im gonna read about bcrypt for the next implementations